docs/howto/distributed-tracing/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,5 @@
+swarm:
+  jaeger:
```

Comment:
great that we are using project-defaults. We should also change example in the examples repo to use it - instead of using jaeger env vars.